esphome/components/wifi/__init__.py

```diff
@@ -397,6 +397,10 @@ def add_sta(ap, network):
             lambda ap: cg.add(var.set_ap(wifi_network(conf, ap, ip_config))),
         )
         cg.add(var.set_ap_timeout(conf[CONF_AP_TIMEOUT]))
+        cg.add_define("ENABLE_WIFI_AP")
```

Comment:
To match other defines in ESPHome we follow `USE_<component>[_<something>]`

```suggestion
        cg.add_define("USE_WIFI_AP")
```

Reply:
Making this change now; should I also add `USE_WIFI_API` to `defines.h` for static analyzer/IDE usage?

Reply:
Yes =)